src/openforms/formio/rendering/tests/test_component_node.py

```diff
@@ -0,0 +1,318 @@
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from openforms.forms.tests.factories import FormFactory
+from openforms.submissions.rendering import Renderer, RenderModes
+from openforms.submissions.tests.factories import (
+    SubmissionFactory,
+    SubmissionStepFactory,
+)
+
+from ..constants import RenderConfigurationOptions
+from ..nodes import ComponentNode
+from ..registry import Registry
+
+# from rest_framework.reverse import reverse
```

Comment:
Maybe forgotten commented out import? :upside_down_face:

Reply:
indeed! I've fixed that in the follow up PR :)